cashLetterControl.go

```diff
@@ -64,7 +64,10 @@ func (clc *CashLetterControl) setRecordType() {
 	}
 
 	clc.recordType = "90"
-	clc.SettlementDate = time.Now()
+	now := time.Now()
+	if now.After(clc.SettlementDate) {
```

Comment:
Would this make more sense as:
```go
if clc.SettlementDate.IsZero() {
    clc.SettlementDate = time.Now()
}
```

Reply:
Ah, great! I've not used Go and didn't realize a time has `.IsZero()` ... though I'd kind of prefer a more explicit notion of null here, but it's at least an improvement in the event there's a use case for backdating (until that one instant in 1970 ;P ).

Reply:
Oh yeah, Go takes a bit of getting used to in this regard. Types like `struct`, `string`, `int`, etc. can't be `nil`. Instead they have _zero values_: https://tour.golang.org/basics/12

In this case it boils down to `time.Time` can't be nil, though a pointer to it could be (`*time.Time`).